### ISSUE TITLE
Identify/isolate the PyTorch version of the model

### ISSUE THREAD

**Issue by dorchard** [m2lines/convection-parameterization-in-CAM#19]
In order to test the Fortran implementation of the NN, it would be great to have the original Python inference code so that we can compare the two for input-output equality. Mentioned in https://github.com/m2lines/convection-parameterization-in-CAM/issues/1

The PyTorch models are in [this file](https://github.com/m2lines/Neural_nework_parameterization/blob/main/NN_training/src/ml_train_nn.py). Unfortunately, there are a number of duplicated models with/without extra layers and batchnorms, etc. Some actions:

- [ ] Get the exact model reference from the paper.
    - From the paper: The  results  presented  in  the  main  paper  are  for  NNs  (both  NN1  and  NN2)  with  five  layers  (four  hidden  layers) with 128 nodes and rectified linear unit (ReLu) activation functions. Results for different NN archi-tectures are shown in the supplementary information (Figure S2).
- [ ] Move the code over to this repo.
- [ ] Find the weights for the Python/Fortran models and figure out how to marry them up.
